tests/AutoReview/DuplicatesTestsTest.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\AutoReview;
+
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tests\TestCase;
+
+/**
+ * @internal
+ *
+ * @coversNothing
+ *
+ * @group auto-review
+ * @group covers-nothing
+ */
+final class DuplicatesTestsTest extends TestCase
```

Comment:
```diff
-DuplicatesTests
+DuplicatedTests
```

Reply:
Will be done in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8508